notebooks/CLT/CLT-006/CLT-006-ComCam.ipynb

```diff
@@ -0,0 +1,338 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    plt.plot((zk_table_30_sec['visit'] - zk_table_30_sec['visit'][0])/3, zk_table_30_sec['fwhm_combined'], label='30 Second Exposures')
</pre><p>Could you add any marker for the data so we see them? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/ts_aos_analysis/pull/35/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>